server.ts

```diff
@@ -0,0 +1,57 @@
+import * as express from 'express';
+import { TodoApp } from './src/todo';
+
+const todo = new TodoApp();
+
+const serv = express();
+serv.use(express.json());
+
+const port = 3000;
+
+serv.get('/', function (req, res) {
+    res.send(todo.listAllToDos());
```

Comment:
Use `res.end` instead to indicate to send and complete the request.  Else the response will never be ended.

Reply:
Actually... since the requirement is JSON you want to use `res.json(...)` here.

Reply:
I'm reading online that res.send() does end the request in addition to sending the data. Is that not true?

Reply:
Just reviewed the [send API reference](https://expressjs.com/en/4x/api.html#res.send) myself.  I don't see any wording in there to indicate that `send` ends the request which makes me think that it won't and that it can be called multiple times (imagine a scenario where it could be called multiple times to send multiple parts of a response).  However when reading the [end API reference](https://expressjs.com/en/4x/api.html#res.end) I see that it mentions the following:

> If you need to respond with data, instead use methods such as res.send() and res.json().

So based on that I would infer that send does implicitly end the response even though it fails to mention this in its own documents.  You would be correct then.